pkg/fanal/analyzer/language/dotnet/packagesprops/packagesprops.go

```diff
@@ -0,0 +1,47 @@
+package packagesprops
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	"golang.org/x/xerrors"
+
+	props "github.com/aquasecurity/go-dep-parser/pkg/nuget/packagesprops"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&packagesPropsAnalyzer{})
+}
+
+const (
+	version             = 1
+	packagesPropsSuffix = "Packages.props" // https://github.com/dotnet/roslyn-tools/blob/b4c5220f5dfc4278847b6d38eff91cc1188f8066/src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs#L39-L40
+)
+
+type packagesPropsAnalyzer struct{}
+
+func (a packagesPropsAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+	parser := props.NewParser()
+	res, err := language.Analyze(types.PackagesProps, input.FilePath, input.Content, parser)
+	if err != nil {
+		return nil, xerrors.Errorf("*Packages.props dependencies analysis error: %w", err)
+	}
+
+	return res, nil
+}
+
+func (a packagesPropsAnalyzer) Required(filePath string, _ os.FileInfo) bool {
+	return strings.HasSuffix(filePath, packagesPropsSuffix)
```

Comment:
it looks like `Packages.props` always uses uppercase for first `P` - https://github.com/dotnet/roslyn-tools/blob/b4c5220f5dfc4278847b6d38eff91cc1188f8066/src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs#L39-L40
We don't need to check filenames for lower case.

Reply:
I just ran an experiment with `packages.props` file and it seems to work fine

Reply:
Okay, thanks!
I reverted changes in https://github.com/aquasecurity/trivy/pull/5605/commits/c60fdf215c984bfc9d32bfd5e293d93f68faf472